recipes/quickfix-cpp/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "40b8984" %}
+
+package:
+  name: quickfix-cpp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/quickfix/quickfix/archive/40b8984af8bf9928d78dfd56bd7952702a8ad877.zip
+  sha256: 76224ee1b4658b5241918f2a117acbd430b3aa671e02c264b1edf1d505269f40
+  patches:
+    - remove_net_test.diff
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - libtool
+    - m4
+    - autoconf
+    - automake
+    - make
+    - libxml2
+  host:
+    - libxml2
+    - openssl
+    - ruby
+    - mysql-server
```

Comment:
Do you really need the server in or only the library?

Reply:
That's a good question - was loosely basing my assumptions on https://github.com/quickfix/quickfix/blob/master/scripts/ubuntu_12/install_deps.sh - will look to eval.

Reply:
based on author's suggested pattern, I'd say the server needs to be there